Traceback (most recent call last):
  File "/data/data/com.termux/files/usr/bin/hwinfo", line 106, in <module>
    main()
  File "/data/data/com.termux/files/usr/bin/hwinfo", line 17, in main
    console.print(get_device_info(hwinfo))
  File "/data/data/com.termux/files/usr/bin/hwinfo", line 69, in get_device_info
    info.append(hwinfo["device"]["vendor"], style=VALUE_STYLE)
KeyError: 'device'
